hphp/runtime/ext/std/ext_std_process.cpp

```diff
@@ -201,7 +203,13 @@ struct ShellExecContext final {
 #endif
     }
     if (m_sig_handler) {
```

Comment:
Right—that last sentence is what I was asking about.  This works _iff_ that's true.  You should either assert that it is, or clarify the behavior when it isn't—i.e., we always override a competing installed sigaction, or we never do, or whatever the appropriate behavior is.